--- a/patch/series
+++ b/patch/series
@@ -158,6 +158,7 @@
 0095-hwmon-mlxreg-fan-Fix-out-of-bounds-read-on-array-fan.patch
 0096-hwmon-mlxreg-fan-Modify-PWM-connectivity-validation.patch
 0097-hwmon-mlxreg-fan-Support-distinctive-names-per-diffe.patch
+0097-2-mlxsw-i2c-Fix-chunk-size-setting.patch
 0098-mlxsw-Use-weak-reverse-dependencies-for-firmware-fla.patch
 0099-mlxsw-core_hwmon-Fix-variable-names-for-hwmon-attrib.patch
 0100-mlxsw-core_thermal-Rename-labels-according-to-naming.patch
@@ -302,6 +303,81 @@
 0300-hwmon-pmbus-Add-support-for-MPS-Multi-phase-mp2855-c.patch
 0301-crypto-ccp-Reject-SEV-commands-with-mismatching-comm.patch
 0302-crypto-ccp-Play-nice-with-vmalloc-d-memory-for-SEV-c.patch
+7001-mctp-Add-MCTP-base.patch
+7002-mctp-Add-base-socket-protocol-definitions.patch
+7003-mctp-Add-base-packet-definitions.patch
+7004-mctp-Add-sockaddr_mctp-to-uapi.patch
+7005-mctp-Add-initial-driver-infrastructure.patch
+7006-mctp-Add-device-handling-and-netlink-interface.patch
+7007-mctp-Add-initial-routing-framework.patch
+7008-mctp-Add-netlink-route-management.patch
+7009-mctp-Add-neighbour-implementation.patch
+7010-mctp-Add-neighbour-netlink-interface.patch
+7011-mctp-Populate-socket-implementation.patch
+7012-mctp-Implement-message-fragmentation-reassembly.patch
+7013-mctp-Add-dest-neighbour-lladdr-to-route-output.patch
+7014-mctp-Allow-per-netns-default-networks.patch
+7015-mctp-Add-MCTP-overview-document.patch
+7016-mctp-remove-duplicated-assignment-of-pointer-hdr.patch
+7017-mctp-Specify-route-types-require-rtm_type-in-RTM_-RO.patch
+7018-mctp-Remove-the-repeated-declaration.patch
+7019-mctp-perform-route-destruction-under-RCU-read-lock.patch
+7020-perf-beauty-Update-copy-of-linux-socket.h-with-the-k.patch
+7021-mctp-Allow-MCTP-on-tun-devices.patch
+7022-mctp-Allow-local-delivery-to-the-null-EID.patch
+7023-mctp-locking-lifetime-and-validity-changes-for-sk_ke.patch
+7024-mctp-Add-refcounts-to-mctp_dev.patch
+7025-mctp-Implement-a-timeout-for-tags.patch
+7026-mctp-Add-tracepoints-for-tag-key-handling.patch
+7027-mctp-Do-inits-as-a-subsys_initcall.patch
+7028-doc-mctp-Add-a-little-detail-about-kernel-internals.patch
+7029-mctp-Set-route-MTU-via-netlink.patch
+7030-mctp-Warn-if-pointer-is-set-for-a-wrong-dev-type.patch
+7031-mctp-Avoid-leak-of-mctp_sk_key.patch
+7032-mctp-Implement-extended-addressing.patch
+7033-i2c-core-smbus-Expose-PEC-calculate-function-for-gen.patch
+7034-dt-bindings-net-New-binding-mctp-i2c-controller.patch
+7035-mctp-unify-sockaddr_mctp-types.patch
+7036-mctp-Be-explicit-about-struct-sockaddr_mctp-padding.patch
+7037-mctp-Return-new-key-from-mctp_alloc_local_tag.patch
+7038-mctp-Add-flow-extension-to-skb.patch
+7039-mctp-Pass-flow-data-flow-release-events-to-drivers.patch
+7040-mctp-unify-extended-addressing-implementation-with-u.patch
+7041-mctp-handle-the-struct-sockaddr_mctp-padding-fields.patch
+7042-mctp-handle-the-struct-sockaddr_mctp_ext-padding-fie.patch
+7043-mctp-i2c-MCTP-I2C-binding-driver.patch
+7044-mctp-i2c-Fix-potential-use-after-free.patch
+7045-mctp-i2c-Fix-hard-head-TX-bounds-length-check.patch
+7046-mctp-remove-unnecessary-check-before-calling-kfree_s.patch
+7047-mctp-Remove-redundant-if-statements.patch
+7048-mctp-Don-t-let-RTM_DELROUTE-delete-local-routes.patch
+7049-mctp-emit-RTM_NEWADDR-and-RTM_DELADDR.patch
+7050-mctp-Remove-only-static-neighbour-on-RTM_DELNEIGH.patch
+7051-mctp-Add-helper-for-address-match-checking.patch
+7052-mctp-Allow-keys-matching-any-local-address.patch
+7053-mctp-Add-SIOCMCTP-ALLOC-DROP-TAG-ioctls-for-tag-cont.patch
+7054-mctp-fix-use-after-free.patch
+7055-mctp-replace-mctp_address_ok-with-more-fine-grained-.patch
+7056-mctp-add-address-validity-checking-for-packet-receiv.patch
+7057-mctp-make-__mctp_dev_get-take-a-refcount-hold.patch
+7058-mctp-Fix-incorrect-netdev-unref-for-extended-addr.patch
+7059-mctp-Fix-warnings-reported-by-clang-analyzer.patch
+7060-mctp-Avoid-warning-if-unregister-notifies-twice.patch
+7061-mctp-Fix-check-for-dev_hard_header-result.patch
+7062-mctp-i2c-correct-mctp_i2c_header_create-result.patch
+7063-mctp-Use-output-netdev-to-allocate-skb-headroom.patch
+7064-mctp-defer-the-kfree-of-object-mdev-addrs.patch
+7065-mctp-prevent-double-key-removal-and-unref.patch
+7066-mctp-Fix-an-error-handling-path-in-mctp_init.patch
+7067-mctp-i2c-don-t-count-unused-invalid-keys-for-flow-re.patch
+7068-mctp-Remove-device-type-check-at-unregister.patch
+7069-net-mctp-add-an-explicit-reference-from-a-mctp_sk_ke.patch
+7070-net-mctp-move-expiry-timer-delete-to-unhash.patch
+7071-net-mctp-hold-key-reference-when-looking-up-a-genera.patch
+7072-net-mctp-mark-socks-as-dead-on-unhash-prevent-re-add.patch
+7073-net-mctp-purge-receive-queues-on-sk-destruction.patch
+7074-kernel-resource-Introduce-request_mem_region_muxed.patch
+7075-i2c-asf-Introduce-MCTP-support-over-ASF-controller.patch
 9002-TMP-fix-for-fan-minimum-speed.patch
 9003-platform-mellanox-Introduce-support-of-Nvidia-L1-tra.patch
 9004-DS-OPT-mlxsw-minimal-Downstream-Disable-ethtool-interface.patch

--- a/patch/kconfig-inclusions
+++ b/patch/kconfig-inclusions
@@ -130,6 +130,9 @@
 CONFIG_PINCTRL_AMD=y
 CONFIG_EDAC_AMD64=m
 CONFIG_AMD_XGBE_DCB=y
+CONFIG_USB_NET_DRIVERS=m
+CONFIG_USB_USBNET=m
+CONFIG_USB_NET_CDCETHER=m
 ###-> mellanox_amd64-end
 # For Cisco 8000
 CONFIG_PHYLIB=m
@@ -160,4 +163,16 @@
 [mellanox-arm64]
 CONFIG_THERMAL_STATISTICS=n
 CONFIG_THERMAL_WRITABLE_TRIPS=y
+CONFIG_SERIAL_AMBA_PL011=y
+CONFIG_SERIAL_AMBA_PL011_CONSOLE=y
+CONFIG_SENSORS_MP2891=m
+CONFIG_MMC_SDHCI_OF_DWCMSHC=m
+CONFIG_VFIO_PLATFORM=m
+CONFIG_MLXBF_GIGE=m
+CONFIG_I2C_MLXBF=m
+CONFIG_GPIO_MLXBF3=m
+CONFIG_PINCTRL_MLXBF3=m
+CONFIG_MLXBF_TMFIFO=m
+CONFIG_MLXBF_BOOTCTL=m
+CONFIG_MLXBF_PTM=m
 
